graphene/types/tests/test_enum.py

```diff
@@ -518,3 +518,18 @@ class Query(ObjectType):
     assert result.data == {"createPaint": {"color": "RED"}}
 
     assert color_input_value == RGB.RED
+
+
+def test_hashable_enum():
+    class RGB(Enum):
+        """Available colors"""
+
+        RED = 1
+        GREEN = 2
+        BLUE = 3
+
```

Comment:
This should also include the instance-creation pattern for enums, to ensure completeness:
https://docs.graphene-python.org/en/latest/types/enums/#definition

```python
Episode = graphene.Enum('Episode', [('NEWHOPE', 4), ('EMPIRE', 5), ('JEDI', 6)])
```

Reply:
Thanks! I added a test in my amended commit.